test/e2e/default-fixture.js

```diff
@@ -110,6 +110,7 @@ function defaultFixture(inputChainId = CHAIN_IDS.LOCALHOST) {
           '__FIXTURE_SUBSTITUTION__currentDateInMilliseconds',
         defaultHomeActiveTabName: null,
         fullScreenGasPollTokens: [],
+        canTrackWalletFundsObtained: true,
```

Comment:
we shouldn't be adding this property here, as this is the default wallet state at init, which uses a very old state (see migration version `FIXTURE_STATE_METADATA_VERSION = 74;`) and at that "time", this property didn't exist in the wallet.

Instead, we should remove this from here, and also from the before-init files.
This will result in not having this property at the beginning (expected) but then having it after the wallet is initialized (expected)
```suggestion
```